nodestream/pipeline/extractors/ttls.py

```diff
@@ -7,13 +7,21 @@
 
 class TimeToLiveConfigurationExtractor(Extractor):
     def __init__(
-        self, graph_object_type: str, configurations: Iterable[Dict[str, Any]]
+        self,
+        graph_object_type: str,
+        configurations: Iterable[Dict[str, Any]],
+        global_expiry_in_hours: int = None,
     ) -> None:
         self.graph_object_type = GraphObjectType(graph_object_type)
         self.configurations = configurations
+        self.global_expiry_in_hours = global_expiry_in_hours
```

Comment:
```suggestion
        override_expiry_in_hours: int = None,
    ) -> None:
        self.graph_object_type = GraphObjectType(graph_object_type)
        self.configurations = configurations
        self.override_expiry_in_hours = override_expiry_in_hours
```

`global`, in my mind, implies that it is set for all types when it actuality only set for types that are provided.